sky/clouds/service_catalog/azure_catalog.py

```diff
@@ -11,6 +12,11 @@
 
 _df = common.read_catalog('azure/vms.csv')
 
+# General-purpose instance with Intel Ice Lake 8370C
```

Comment:
Add: This is the latest general-purpose instance family as of Jan 2023.

Reply:
Added.